cmd/copy.go

```diff
@@ -1053,7 +1053,8 @@ func (cca *cookedCopyCmdArgs) processCopyJobPartOrders() (err error) {
 
 	// For OAuthToken credential, assign OAuthTokenInfo to CopyJobPartOrderRequest properly,
 	// the info will be transferred to STE.
-	if cca.credentialInfo.CredentialType == common.ECredentialType.OAuthToken() {
+	if cca.credentialInfo.CredentialType == common.ECredentialType.OAuthToken() ||
+		cca.credentialInfo.CredentialType == common.ECredentialType.AutoLogin() {
```

Comment:
I sort of convinced myself that the new enum is just noise. All we're doing is OAuth, except for persisting on disk. Enough logging should be okay instead of repeating case statements twice for OAuth and Auto all through the code.